src/aggregator.rs

```diff
@@ -13,6 +13,12 @@ pub trait Aggregator<Candle, T: TakerTrade> {
     /// Some output only when a new candle has been created,
     /// otherwise it returns None
     fn update(&mut self, trade: &T) -> Option<Candle>;
+
+    /// Close current active candle and return it
+    /// 
+    /// # Returns:
+    /// Last active candle, or None if candle was already finalized
+    fn close(&mut self) -> Option<Candle>;
```

Comment:
```suggestion

    /// Get a reference to an unfinished `Candle`.
    /// Accessing a `Candle` using this method does not guarantee that the `AggregationRule` is respected.
    /// It is generally advised to call `update` instead and use the resulting `Candle` if its `Some`.
    fn candle(&self) -> &Candle;
```

Reply:
I agree, that's what I ended up doing - implemented my own aggregator with method like this. Will update the PR